src/store/classificationRequirements/index.ts

```diff
@@ -48,6 +55,39 @@ export class ClassificationRequirementsStore extends VuexModule {
       this.sessionProperties,
       ATAT_CLASSIFICATION_LEVELS_KEY
     );
+
+  }
+
+  @Mutation
+  public async setCurrentENVClassificationLevels(value: ClassificationLevelDTO[]): Promise<void> {
+    this.currentEnvClassificationLevels = value;
+    storeDataToSession(
+      this,
+      this.sessionProperties,
+      ATAT_CLASSIFICATION_LEVELS_KEY
+    );
+
+  }
+
+  @Mutation
+  public async setEnvironmentInstance(value: EnvironmentInstanceDTO[]): Promise<void> {
```

Comment:
pls delete this mutation - it was removed when current env classification requirements was refactored to be in the CurrentEnvironment store